localflavor/md/validators.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import re
+
+from django.core.exceptions import ValidationError
+from django.core.validators import RegexValidator
+from django.utils.translation import ugettext_lazy as _
+
+
+class MDIDNOFieldValidator(RegexValidator):
+    """
+    Validation for Moldavian IDNO.
+
+    .. versionadded:: 1.6
+    """
+
+    error_message = _('Enter a valid IDNO number.')
+
+    def __init__(self):
+        super(MDIDNOFieldValidator, self).__init__(regex='^\d{13}$',
+                                                   message=self.error_message)
+
+    def __call__(self, value):
+        super(MDIDNOFieldValidator, self).__call__(value)
+
+
+class MDLicensePlateValidator(RegexValidator):
+    """
+    Validation for Moldavian License Plates.
+
+    .. _Registration Plates of Moldova:
+       https://en.wikipedia.org/wiki/Vehicle_registration_plates_of_Moldova
+
+    .. versionadded:: 1.6
+    """
+
+    error_message = _('Enter a valid license plate.')
+    # regex = '^\d{9}$'
+    # message = error_message
+
+    def __init__(self):
+        super(MDLicensePlateValidator, self).__init__(regex='^\d{9}$',
```

Comment:
Same comment as above: you should override `regex` and `message` instead of setting them in `__init__`. Or was that not working?

The commented code should be removed as well.